app/controllers/application_controller.rb

```diff
@@ -1215,6 +1216,12 @@ def get_view(db, options = {}, fetch_data = false)
       params[:sortby]      = view.headers.index(params[:sort_choice])
     end
 
+    report_symbols = [:all_sortcol, :savedreports_tree_sortcol, :reports_tree_sortcol]
+    # Check if the symbol representing the page is included in the array above and then check if the variable for the sort column (session[sortcol_sym]) is nil
+    if report_symbols.include?(sortcol_sym) && session[sortcol_sym].nil?
+      session[sortcol_sym] = ReportController::DEFAULT_SORT_COL
+      session[sortdir_sym] = "DESC"
```

Comment:
This might also make sense to move into a constant like `ReportController::DEFAULT_SORT_DIR`, then you can also use it on line 127 in app/controllers/report_controller/reports.rb and 146 in app/controllers/report_controller/saved_reports.rb